src/test/java/de/caritas/cob/mailservice/api/service/TranslationServiceTest.java

```diff
@@ -0,0 +1,70 @@
+package de.caritas.cob.mailservice.api.service;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.test.util.ReflectionTestUtils;
+
+import de.caritas.cob.mailservice.api.model.Dialect;
+import de.caritas.cob.mailservice.config.apiclient.TranslationManagementServiceApiClient;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoInteractions;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class TranslationServiceTest {
+
+  private TranslationService translationService;
+
+  @Mock
+  private TranslationManagementServiceApiClient translationManagementServiceApiClient;
+
+  @Mock
+  private DefaultTranslationsService defaultTranslationsService;
+
+  public final String TRANSLATION_MANAGEMENT_SYSTEM_ENABLED_FIELD_NAME = "translationManagementSystemEnabled";
+
+  @BeforeEach
+  public void setUp() {
+    translationService = new TranslationService(translationManagementServiceApiClient, defaultTranslationsService);
+  }
+
+  @Test
+  void fetchTranslations_Should_call_tryFetchTranslationsFromTranslationManagementService_When_translationManagementSystemEnabled_is_true() {
```

Comment:
I would rename this test not to be that code oriented. Should_fetchTranslationsFromAPI_When_...